docs/docs/04-standard-library/01-cloud/queue.md

```diff
@@ -51,9 +51,7 @@ let q = new cloud.Queue();
 
 new cloud.Function(inflight () => {
   q.push("message a");
-  q.push("message b");
-  q.push("message c");
-  q.push("message d");
+  q.push("message b", "message c", "message d");
```

Comment:
doc updates for sdk resources need to be made in the SDK itself: https://github.com/winglang/wing/blob/main/libs/wingsdk/src/cloud/queue.md

Reply:
Right, forgot about that change.